src/resolvers/projectResolver.ts

```diff
@@ -1519,6 +1534,55 @@ export class ProjectResolver {
     }
   }
 
+  @Query(returns => ProjectUpdatesResponse, { nullable: true })
+  async projectUpdates(
+    @Arg('take', type => Int, { defaultValue: 10 }) take: number,
+    @Arg('skip', type => Int, { defaultValue: 0 }) skip: number,
+    @Ctx() { req: { user } }: MyContext,
+  ): Promise<ProjectUpdatesResponse> {
+    const latestProjectUpdates = await ProjectUpdate.query(`
+      SELECT pu.id, pu."projectId"
+      FROM project_update as pu
+      WHERE pu.id = (
+        SELECT puu.id
+        FROM project_update as puu
+        WHERE puu."isMain" = false AND pu."projectId" = puu."projectId"
+        ORDER BY puu."createdAt" DESC
+        LIMIT 1
+      )
+      ORDER BY pu."createdAt" DESC
+      LIMIT ${take}
+      OFFSET ${skip}
+    `);
+
```

Comment:
I ran into this issues with distinctOn and joins, no amount of orderbys or groupbys to fix the error worked, and even if they did, it was returning not desired results. 

So I separated it to two queries for it to work.